text-to-speech/src/main/java/com/ibm/watson/text_to_speech/v1/TextToSpeech.java

```diff
@@ -154,6 +160,53 @@ public TextToSpeech(String serviceName, Authenticator authenticator) {
     this.configureService(serviceName);
   }
 
+  /**
+   * Synthesize audio.
+   *
+   * <p>Synthesizes text to audio that is spoken in the specified voice. The service bases its
+   * understanding of the language for the input text on the specified voice. Use a voice that
+   * matches the language of the input text.
+   *
+   * <p>The method accepts a maximum of 5 KB of input text in the body of the request, and 8 KB for
+   * the URL and headers. The 5 KB limit includes any SSML tags that you specify. The service
+   * returns the synthesized audio stream as an array of bytes.
+   *
+   * <p>### Audio formats (accept types)
+   *
+   * <p>For more information about specifying an audio format, including additional details about
+   * some of the formats, see [Audio
+   * formats](https://cloud.ibm.com/docs/text-to-speech?topic=text-to-speech-audioFormats#audioFormats).
+   *
+   * @param synthesizeOptions the {@link SynthesizeOptions} containing the options for the call
+   * @param callback the {@link SynthesizeCallback} callback
+   * @return a {@link WebSocket} instance
+   */
+  public WebSocket synthesizeUsingWebSocket(
```

Comment:
You need to add the new properties `ratePercentage` and `pitchPercentage` to `TextToSpeechWebSocketListener.java`